osu.Game/Overlays/Mods/ModSelectColumn.cs

```diff
@@ -73,6 +80,16 @@ protected ModSelectColumn()
                                     RelativeSizeAxes = Axes.X,
                                     Height = header_height + ModSelectPanel.CORNER_RADIUS
                                 },
+                                triangles = new TrianglesV2
+                                {
+                                    Anchor = Anchor.TopRight,
+                                    Origin = Anchor.TopRight,
+                                    RelativeSizeAxes = Axes.X,
+                                    Width = 1.1f,
+                                    Height = header_height,
+                                    Shear = new Vector2(-ShearedOverlayContainer.SHEAR, 0),
+                                    Masking = true
```

Comment:
```suggestion
                                    Velocity = 0.7f,
                                    Masking = true
```

I think slowing the triangles down feels better. They were zooming quite fast..